Formula/libtiff.rb

```diff
@@ -15,8 +15,10 @@ class Libtiff < Formula
 
   option :universal
   option :cxx11
+  option "with-lzma", "Include support for LZMA compression"
```

Comment:
You're right, LZMA is a compression algorithm and there are multiple projects implementing it, though most distributions and package managers nowadays tend to default to the implementation provided by the `xz` project. I was a bit confused here. Thanks for the clarification!